js/packages/components/chat/message/user-message/user-message-box/FileMessage.tsx

```diff
@@ -65,7 +65,7 @@ export const FileMessage: React.FC<{
 					},
 				]}
 			>
-				{medias[0].filename}
+				{medias[0].filename?.substring(0, 20).concat('...')}
```

Comment:
why not use the "ellipsize" feature of react-native text ?

Reply:
Yep i can use it 👍